chipy_org/templates/homepage.html

```diff
@@ -132,7 +132,12 @@ <h4>{{ announcement.headline }}</h4>
 
         <div class="row-fluid">
             <div class="module span6">
-                <h3>In the Loop</h3>
+                {% if next_meeting.custom_title %}
+                    <h3>{{ next_meeting.custom_title }}</h3>
+                {% else %}        
+                    <h3>In the Loop</h3>
+                {% endif %}
```

Comment:
I don't see any reason why this wouldn't work, but why not just use `next_meeting.title`?

Reply:
next_meeting refers to when meeting_type is null. Since it is null and not an object, it can not have a 'title' field.  

If you want to say 'next_meeting.title', the user would have to make a 'meeting_type' named 'main meeting'. Then I could add a 'title' field to that object.  I'd also have to change the code in `/chip_org/apps/meetings/views.py` under the InitialRSVPMixin class.

Currently the code is: 

https://github.com/chicagopython/chipy.org/blob/master/chipy_org/apps/meetings/views.py#L45
```
class InitialRSVPMixin():
    def get_next_main_meeting(self):
        return (Meeting.objects
                .filter(meeting_type__isnull=True)
                .filter(when__gt=datetime.datetime.now()-datetime.timedelta(hours=6))
                .order_by('when')
                .first())
```

Reply:
Ok, I just read your comment from way above. I wasn't reading your comments in order! 

> If you want, you could make this final return the default for the main meeting as not custom title and no meeting type is a main meeting with a default title. I'd put whatever is in the template now.

So if I understand your comment correctly, we could define a default title for the main meeting here. 
> return "" # return an empty string if neither title is defined

And then yes, I can change the template to be 'next_meeting.title' without having to change any other code.

Reply:
Originally, I wrote the code that way because next_meeting in this case refers to a meeting_type of null. And null can't have a 'default_title' since it isn't an object. 

https://github.com/chicagopython/chipy.org/blob/master/chipy_org/apps/meetings/views.py

```class InitialRSVPMixin():
    def get_next_main_meeting(self):
        return (Meeting.objects
                .filter(meeting_type__isnull=True)
                .filter(when__gt=datetime.datetime.now()-datetime.timedelta(hours=6))
                .order_by('when')
                .first())
```
Now with your suggestion, I can just define the 'default_title' as 'In the Loop'.  Nice catch!